### ISSUE TITLE
Automatically add homekit attribute

### ISSUE THREAD

**Issue by joshuata** [adrianByv/home-assistant-custom-components-switch-humidifier#2]
It would be nice to automatically add the `linked_humidity_sensor` attribute to the homekit config, especially since it can't be done if HomeKit is configured using the frontend. I am looking into whether this can be done currently, but it might require changes to HomeAssistant Core

**Comment by tzungtzu** [comment]
yes, the <linked_humidity_sensor> could not set from the frontend.
